src/main/java/unbanner/StudentController.java

```diff
@@ -19,21 +20,23 @@ public String StudentsList(Model model) {
 
     @RequestMapping("/student/{id}")
 
+    //show one student
```

Comment:
More documentation.

Which one student is showed? What is the format of the lookup string?